src/test/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/resource/CreateCsipRecordsIntTest.kt

```diff
@@ -192,45 +192,6 @@ class CreateCsipRecordsIntTest : IntegrationTestBase() {
     }
   }
 
-  @Test
-  fun `400 bad request - incident involvement not found`() {
```

Comment:
I can see int tests for contributory factor code(s) being missing or inactive but not for incidentTypeCode, incidentLocationCode and refererAreaCode. Have I missed those or are they tested elsewhere?